rootston/seat.c

```diff
@@ -260,6 +260,7 @@ static void roots_drag_icon_handle_surface_commit(struct wl_listener *listener,
 		void *data) {
 	struct roots_drag_icon *icon =
 		wl_container_of(listener, icon, surface_commit);
+	roots_drag_icon_update_position(icon);
 	roots_drag_icon_damage_whole(icon);
```

Comment:
No need to damage after `roots_drag_icon_update_position`, this is already done in `roots_drag_icon_update_position`